internal/api/shared/authorization/permission.go

```diff
@@ -0,0 +1,44 @@
+// Copyright 2021 The Perses Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package authorization
+
+import (
+	v1 "github.com/perses/perses/pkg/model/api/v1"
+)
+
+const (
+	wildcard = "*"
```

Comment:
like you did for the action, you should have a constant `WildcardScope` in the package `model/api/v1`

probably you will need to create a special  `Kind`  for the scope to include the wildcard